commands/peers/peer-rpc.proto

```diff
@@ -24,11 +24,9 @@ message PeerDeleteReq {
 }
 
 message EtcdConfigReq {
- string PeerName = 1;
- string Name = 2;
+ string EtcdName = 2;
```

Comment:
We haven't yet finalized these protobuf messaages. So we can still change the tags of message members. Start from 1 again. If we were changing after we had finalized, we'd have to give new tags to any new member, like `EtcdName` here, and retire the old member and tag.

Refer https://developers.google.com/protocol-buffers/docs/proto3#assigning-tags for more info.